cassiopeia/tools/fitness_estimator/_lbi_jungle.py

```diff
@@ -0,0 +1,116 @@
+import os
+import sys
+import tempfile
+from typing import Optional
+
+import networkx as nx
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+print(f"dir_path = {dir_path}")
+sys.path.append(os.path.join(dir_path, "_jungle"))
+import jungle as jg
```

Comment:
This is horrible but I didn't know a better way to do it.

Reply:
Ha, yeah this isn't pretty but unless we make jungle a module (adding an `__init__.py` etc) then we can't import it directly, right? We can keep this here as is for now.

Reply:
I think I had originally tried with `__init__.py` but it didn't work for some reason. So I just used this. Agree it's fine to keep as is.